src/idl_gen_python.cpp

```diff
@@ -38,6 +40,43 @@ static std::string TypeName(const FieldDef &field);
 // Hardcode spaces per indentation.
 const std::string Indent = "    ";
 
+const std::string keywords_[] = {
+  "False",
+  "None",
+  "True",
+  "and",
+  "as",
+  "assert",
+  "break",
+  "class",
+  "continue",
+  "def",
+  "del",
+  "elif",
+  "else",
+  "except",
+  "finally",
+  "for",
+  "from",
+  "global",
+  "if",
+  "import",
+  "in",
+  "is",
+  "lambda",
+  "nonlocal",
+  "not",
+  "or",
+  "pass",
+  "raise",
+  "return",
+  "try",
+  "while",
+  "with",
+  "yield"
+};
+const std::unordered_set<std::string> Keywords(std::begin(keywords_), std::end(keywords_));
```

Comment:
Would be better to stick this in `PythonGenerator` so it is only initialized when used.